iot-e2e-tests/docker/readme.md

```diff
@@ -0,0 +1,35 @@
+# Docker for WSL setup for local test
+
+This script will setup docker images and run instances of these images to support E2E testing.
+
+Before running docker-setup.sh script:
+
+- Install WSL (See installation details here: https://docs.microsoft.com/windows/wsl/install). Reboot after install.
+
+- Install Docker on WSL (For docker on Ubuntu, see installation details here: https://docs.docker.com/engine/install/ubuntu/#install-using-the-repository, see installation details for other Linux distribution here: https://docs.docker.com/engine/install/). Reboot after install.
```

Comment:
nit: for consistency, would it be worth using hyperlinks?

Reply:
Good idea